Logger/BaseLoggerMixins.cs

```
@@ -1,6 +1,45 @@
-﻿namespace Logger;
-
-public static class BaseLoggerMixins
-{
-
-}
+﻿namespace Logger;
+using System.Globalization;
+#nullable enable
+
+public static class BaseLoggerMixins
+{
+
+    public static void Error(this BaseLogger? logger, string message, params object[] arguments)

```

Why is logger nullable? I expect to call this message, a logger is needed.

---

I did this to resolve: 

"These methods should throw an exception if the BaseLogger parameter is null. There are a couple example unit tests to get you started."

I think this is correct since it says if the baseLogger is null throw an exception and you would need to input null for this to happen, though I maybe miss understanding.  I will ask in class tommorrow.